frontend/src/metabase-types/api/database.ts

```diff
@@ -22,20 +24,25 @@ export type DatabaseFeature =
   | "nested-queries"
   | "standard-deviation-aggregations"
   | "persist-models"
-  | "persist-models-enabled";
+  | "persist-models-enabled"
+  | "set-timezone";
 
 export interface Database extends DatabaseData {
   id: DatabaseId;
-  is_saved_questions: boolean;
+  is_saved_questions?: boolean;
```

Comment:
Why this has been changed to optional? I believe it would be always returned by the backend?